src/lib/Execute.sol

```diff
@@ -6,18 +6,18 @@ import {Address} from "openzeppelin-contracts/utils/Address.sol";
 abstract contract Execute {
     event Executed(address indexed executor, address indexed to, uint256 value, bytes data);
 
-    function execute(address to, uint256 value, bytes calldata data) public returns (bytes memory executeData) {
-        _checkCanExecute();
-        (address guard, bytes memory checkBeforeData) = _beforeExecute(to, value, data);
+    function call(address to, uint256 value, bytes calldata data) public returns (bytes memory executeData) {
```

Comment:
can we rename to `executeCall`? I'd like to avoid ambiguities with solidity's native `.call()` method

Reply:
Resolved in 6d3da8c